spec/components/layout_super_navigation_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+
+describe "Super navigation header", type: :view do
+  def component_name
+    "layout_super_navigation"
+  end
+
+  it "renders the super navigation header" do
+    render_component({})
+
+    assert_select ".gem-c-super-navigation-header"
+  end
```

Comment:
Good idea - I've updated the tests.

I think checking to make sure that `aria-labelledby` is a good test - but since this component relies on the translation file to populate the menu it could be a bit flaky. To fix this I'd like to refactor the component slightly to accept parameters for the menu that would overwrite the defaults, which should make it easier to test - but I think that's outside the scope of this particular pull request.